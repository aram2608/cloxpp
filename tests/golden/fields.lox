class Bag {}
var bag = Bag();
bag.item = "apple";
print bag.item; // expect "apple"