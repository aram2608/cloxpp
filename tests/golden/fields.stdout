Token: CLASS class 1
Token: LEFT_BRACE { 1
Token: RIGHT_BRACE } 1
Token: VAR var 2
Token: EQUAL = 2
Token: LEFT_PAREN ( 2
Token: RIGHT_PAREN ) 2
Token: SEMICOLON ; 2
Token: DOT . 3
Token: IDENTIFIER item 3
Token: EQUAL = 3
Token: STRING "apple" 3
Token: SEMICOLON ; 3
Token: PRINT print 4
Token: DOT . 4
Token: IDENTIFIER item 4
Token: SEMICOLON ; 4
Token: EOF  4
